File: 2678-number-of-senior-citizens/2678-number-of-senior-citizens.py
class Solution:
    def countSeniors(self, details: List[str]) -> int:  
       res=[]
       for i in details:
          x=i[11:13]
          v=int(x)
          res.append(v)
          p=filter(lambda x: x >60,res)
       return len(list(p))

File: 2678-number-of-senior-citizens/2678-number-of-senior-citizens.cpp
class Solution {
public:
    int countSeniors(vector<string>& details) {
        int res=0;
        for(auto i :details)
        {
            int str=stoi(i.substr(11,2));
                res+=str>60;
        }
       return res; 
    }
};